lib/manager/gomod/__snapshots__/artifacts.spec.ts.snap

```diff
@@ -275,15 +275,19 @@ Array [
     },
   },
   Object {
-    "cmd": "docker run --rm --name=renovate_go --label=renovate_child -v \\"/tmp/github/some/repo\\":\\"/tmp/github/some/repo\\" -v \\"/tmp/renovate/cache\\":\\"/tmp/renovate/cache\\" -e GOPROXY -e GOPRIVATE -e GONOPROXY -e GONOSUMDB -e GOFLAGS -e CGO_ENABLED -e GIT_CONFIG_KEY_0 -e GIT_CONFIG_VALUE_0 -e GIT_CONFIG_COUNT -w \\"/tmp/github/some/repo\\" renovate/go:latest bash -l -c \\"go get -d ./...\\"",
+    "cmd": "docker run --rm --name=renovate_go --label=renovate_child -v \\"/tmp/github/some/repo\\":\\"/tmp/github/some/repo\\" -v \\"/tmp/renovate/cache\\":\\"/tmp/renovate/cache\\" -e GOPROXY -e GOPRIVATE -e GONOPROXY -e GONOSUMDB -e GOFLAGS -e CGO_ENABLED -e GIT_CONFIG_KEY_0 -e GIT_CONFIG_VALUE_0 -e GIT_CONFIG_KEY_1 -e GIT_CONFIG_VALUE_1 -e GIT_CONFIG_KEY_2 -e GIT_CONFIG_VALUE_2 -e GIT_CONFIG_COUNT -w \\"/tmp/github/some/repo\\" renovate/go:latest bash -l -c \\"go get -d ./...\\"",
     "options": Object {
       "cwd": "/tmp/github/some/repo",
       "encoding": "utf-8",
       "env": Object {
         "CGO_ENABLED": "1",
-        "GIT_CONFIG_COUNT": "1",
-        "GIT_CONFIG_KEY_0": "url.https://some-token@github.com/.insteadOf",
+        "GIT_CONFIG_COUNT": "3",
+        "GIT_CONFIG_KEY_0": "url.\\\"https://api:some-token@github.com/\\\".insteadOf",
+        "GIT_CONFIG_KEY_1": "url.\\\"https://ssh:some-token@github.com/\\\".insteadOf",
+        "GIT_CONFIG_KEY_2": "url.\\\"https://git:some-token@github.com/\\\".insteadOf",
```

Comment:
```suggestion
        "GIT_CONFIG_KEY_1": "url.\\\"https://ssh:some-token@github.com/\\\".insteadOf",
        "GIT_CONFIG_KEY_2": "url.\\\"https://git:some-token@github.com/\\\".insteadOf",
```

Does this actually work? e.g. are `ssh` and `git` really valid usernames?

Reply:
Yes, the username is not important, the token is the important part. The username is arbitrary.

Reply:
Is that a GitHub specific "feature"?

Reply:
I found no mentioning of this. 
According to the [link](https://coolaj86.com/articles/vanilla-devops-git-credentials-cheatsheet/) this should work on other platforms as well, but I cannot guarantee it.